test/Dapr.Actors.Test/ActorProxyTests.cs

```diff
@@ -40,8 +41,29 @@ public void TestCreateActorProxyThatImplementsInterface_Success()
         public void TestCreateActorProxyThatImplementsInterface_NonSuccess()
         {
             var actorId = new ActorId("abc");
-            Action action = () =>  ActorProxy.Create(actorId, typeof(ActorId), "TestActor");
+            Action action = () => ActorProxy.Create(actorId, typeof(ActorId), "TestActor");
             action.Should().Throw<ArgumentException>();
         }
+
+        [Fact]
+        public void TestCustomSerializerOptionsOnDefaultActorProxyFactory_Success()
+        {
+            var factory = new ActorProxyFactory();
+            factory.DefaultOptions.JsonSerializerOptions = new JsonSerializerOptions();
+
+            var actorId = new ActorId("abc");
+            var proxy = ActorProxy.Create(actorId, typeof(ITestActor), "TestActor");
+
+            Assert.NotNull(proxy);
```

Comment:
Can we verify that the options actually got to the proxy? Should the serialization options be an internal property on the proxy so we can check it here?

Reply:
Yeah I was trying to think of how we can go about testing that. Making the options into an internal property seems like it will fit the bill so I'll go ahead with that.